products/al2023/profiles/cis_server_l1.profile

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+metadata:
+    version: 1.0.0
+    SMEs:
+        - hipponix
+
+reference: https://www.cisecurity.org/benchmark/amazon_linux/
+
+title: 'CIS Amazon Linux 2023 Benchmark for Level 1 - Server'
+
+description: |-
+    This profile defines a baseline that aligns to the "Level 1 - Server"
+    configuration from the Center for Internet Security® Amazon Linux
+    2023 Benchmark™, v1.0.0, released 2023-06-26.
+
+    This profile includes Center for Internet Security®
+    Amazon Linux 2023 CIS Benchmarks™ content.
+
+selections:
+    - cis_al2023:all:l1_server
+    - '!file_ownership_home_directories'
```

Comment:
Same as above.

If you don't want these rules in the profile you can remove them from the control.